packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -354,6 +355,14 @@ class SnackBar extends StatefulWidget {
   ///       right padding.
   final EdgeInsetsGeometry? padding;
 
+  /// Additional constraints to apply to the child.
+  ///
+  /// The constructor `width` and `height` arguments are combined with the
```

Comment:
It seems the documentation here is same as the `Container.constraints`. However `SnackBar` constructor has no `height` property, so it probably doesn't make sense to mention it here. 

Also based on `Snackbar`'s documentation, the `SnackBar.width` can only be used when `SnackBar.behavior` is `floating`. It would be better if we can mention how the `constraints` will work with `width` when `width` is not null. Currently looks like the `constraints` will just be ignored when `width` is not null, we can mention this behavior in documentation. Or maybe another option is to add `constraints: constraints` in [the `Container` when `width` is not null](https://github.com/flutter/flutter/blob/e1702a96f679772847459650670bbe9f04480840/packages/flutter/lib/src/material/snack_bar.dart#L744), but it might cause `Snackbar` not centered when there is a conflict between `width` and `constraints`.

Reply:
I've tried to describe it. I'm not very good at it.
I've also added tests to show that `minWidth` and `maxWidth` are ignore